src/app/app-routing.module.ts

```diff
@@ -3,14 +3,16 @@ import { RouterModule, Routes } from '@angular/router';
 import { AppComponent } from './app.component';
 import { IndexComponent } from './components/index/index.component';
 import { ItinerarylistComponent } from './components/itinerary/itinerarylist/itinerarylist.component';
+import { MyItinirariesComponent } from './components/itinerary/my-itiniraries/my-itiniraries.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7157930)
```suggestion
import { MyItinirariesComponent } from "./components/itinerary/my-itiniraries/my-itiniraries.component";
```